RELEASES.md

```diff
@@ -40,37 +40,52 @@ branches tags. See [#6072](https://github.com/tendermint/tendermint/pull/6072)
 for more context.
 
 In the following example, we'll assume that we're making a backport branch for
-the 0.35.x line.
+the 0.38.x line.
 
 1. Start on `main`
 
-2. Create and push the backport branch:
+2. Ensure that there is a [branch protection
```

Comment:
Can we have a similar rule for backport branches as we have for feature branches?
So they are automatically protected upon creation

Reply:
It's hard to do that because the "required" tests differ sometimes from version to version.

Right now we have a rule that covers v0.35 through v0.39, but when we hit v0.40 we'll need a new branch protection rule. We also have other rules that cover other ranges of earlier versions, but each of them has slightly different requirements due to the different tests on those branches.

Reply:
Still, we'd need to then have different rules for different sets of branches as the required tests change.